dist/query_parameter_ctrl.ts

```diff
@@ -142,17 +142,20 @@ angular.module('grafana.controllers').controller('GoogleStackdriverQueryParamete
   }
 
   $scope.getGroupByFieldsSegments = function () {
+    let filter = 'metric.type = "' + $scope.target.metricType + '"';
+    if ($scope.target.filter) {
+      filter += ' AND ' + $scope.target.filter;
+    }
```

Comment:
Depending on the version of typescript you're using, you may be able to use string interpolation:
```
let filter = `metric.type = "${scope.target.metricType}"`;
```
I'm not sure how this `$scope` variable works, though.

Reply:
Definitely more readable, done.